apps/condo/domains/organization/schema/ResetOrganizationService.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Generated by `createservice organization.ResetOrganizationService --type mutations`
+ */
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT, INTERNAL_ERROR } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { BillingIntegrationOrganizationContext } = require('@condo/domains/billing/utils/serverSchema')
+const { DV_VERSION_MISMATCH } = require('@condo/domains/common/constants/errors')
+const { loadListByChunks } = require('@condo/domains/common/utils/serverSchema')
+const { B2BAppContext } = require('@condo/domains/miniapp/utils/serverSchema')
+const access = require('@condo/domains/organization/access/ResetOrganizationService')
+const { DELETED_ORGANIZATION_NAME } = require('@condo/domains/organization/constants/common')
+const { Organization, OrganizationLink, OrganizationEmployee } = require('@condo/domains/organization/utils/serverSchema')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const errors = {
+    DV_VERSION_MISMATCH: {
+        mutation: 'resetOrganization',
+        variable: ['data', 'dv'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Unsupported value for dv',
+    },
+    SENDER_MISMATCH: {
+        mutation: 'resetOrganization',
+        variable: ['data', 'sender'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Unsupported value for sender',
+    },
+    ORGANIZATION_NOT_FOUND: {
+        mutation: 'resetOrganization',
+        variable: ['data', 'organizationId'],
+        code: BAD_USER_INPUT,
+        type: 'ORGANIZATION_NOT_FOUND',
+        message: 'Could not find organization by provided id',
+    },
+}
+
+const ResetOrganizationService = new GQLCustomSchema('ResetOrganizationService', {
+    types: [
+        {
+            access: true,
+            type: 'input ResetOrganizationInput { dv: Int! sender: JSON! organizationId: ID! }',
+        },
+        {
+            access: true,
+            type: 'type ResetOrganizationOutput { status: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canResetOrganization,
+            schema: 'resetOrganization(data: ResetOrganizationInput!): ResetOrganizationOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { data: { dv, sender, organizationId } } = args
+
+                if (dv !== 1) {
```

Comment:
it's better to use `const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')`. Look at example from `ConfirmPhoneActionService`

Reply:
Probably it's better to fix the same in `resetUser`